_data/projects/os_100days.yml

```
@@ -0,0 +1,16 @@
+name: OS_100days
+desc: "Ever do the 100daysofcode.com challenge and find yourself skipping, forgetting which day, etc? Well this project aims to correct that with friendly reminders, a simple way to keep track of what day you are on, and will allow you to tweet directly from it. (hopefully)"
+site: "https://discord.gg/AntJA73Qx2"
+tags:
+  - javascript 
+  - html5 
+  - css3 
+  - help-wanted 
+  - first-timers 
+  - up-for-grabs 
+  - hactoberfest 
+  - first-contributions 
+  - hacktoberfest2022
+upforgrabs:
+  name: "coding with drew"
+  link: https://github.com/drewlearns/OS_100days/issues
```

:art: let's deep link to the issue list here for convenience

```suggestion
  link: https://github.com/drewlearns/OS_100days/labels/coding%20with%20drew
```